client/asset/dcr/dcr.go

```diff
@@ -808,23 +828,57 @@ func (dcr *ExchangeWallet) Connect(ctx context.Context) (*sync.WaitGroup, error)
 	return &wg, nil
 }
 
+// Reconfigure attempts to reconfigure the wallet.
+func (dcr *ExchangeWallet) Reconfigure(ctx context.Context, cfg *asset.WalletConfig, currentAddress string) (restart bool, err error) {
+	dcrCfg := new(walletConfig)
+	_, err = loadConfig(cfg.Settings, dcr.network, dcrCfg)
+	if err != nil {
+		return false, err
+	}
+
```

Comment:
In `NewWallet`, we're doing 
```Go
// dcrwallet doesn't accept an empty string as an account name. Use the
// default value.
if walletCfg.PrimaryAccount == "" {
	walletCfg.PrimaryAccount = defaultAcctName
}
```

I had used this feature when creating my Decred wallet, and so when I went to reconfigure, I got an error `-11: account not found`.